alica_engine/include/engine/collections/RobotProperties.h

```diff
@@ -17,7 +17,7 @@ struct AgentAnnouncement;
 class RobotProperties
 {
 public:
-    RobotProperties(const PlanRepository& planRepository, const std::string& defaultRole, const AgentAnnouncement& aa);
+    RobotProperties(const PlanRepository&, const std::string& defaultRole, const AgentAnnouncement& aa);
```

Comment:
If this is not needed, can we just remove the arg from the constructor and places that call it?